src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -55,6 +57,7 @@ internal ArrayList(bool trash)
         // Constructs a ArrayList. The list is initially empty and has a capacity
         // of zero. Upon adding the first element to the list the capacity is
         // increased to _defaultCapacity, and then increased in multiples of two as required.
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the default initial capacity.</summary>
```

Comment:
As you noted in your PR comments, all of the ```//``` and ```///``` comments will need to be rationalized before they're merged.